components/clarinet-cli/src/devnet/start.rs

```diff
@@ -29,17 +27,6 @@ pub fn start(
     ),
     String,
 > {
-    let hooks = match load_chainhooks(
```

Comment:
This is used to load chainhook files from the user's directory and to auto-register them with the devnet. This is a useful feature that we want to keep.